src/main/kotlin/io/snyk/plugin/services/SnykAnalyticsService.kt

```diff
@@ -145,6 +147,22 @@ class SnykAnalyticsService : Disposable {
         }
     }
 
+    fun logQuickFixIsDisplayed(event: QuickFixIsDisplayed) {
+        if (!settings.usageAnalyticsEnabled) return
```

Comment:
Is `userId.isBlank()` check needed here or userId is always initialised to that point?

Reply:
It is already initialized - you can't get a quickfix without results, and no results without token. And once we get the token, the userID is updated.